src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -72,9 +72,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
    if (userStore.isUserRegistered(username)) {
      User user = userStore.getUser(username);
-     if(BCrypt.checkpw(password, user.getPassword())) {
-       request.getSession().setAttribute("user", username);
-       response.sendRedirect("/conversations");
+     Boolean isAdmin = user.getIsAdmin();
```

Comment:
You never do anything with this variable. Why is it declared? Does it need to be Boolean or can it be boolean?